### PR TITLE
Game flow

### PR DESCRIPTION
**Pull request by teelu**
Please take a look at this branch instead of Moves (my 3rd PR)

### CODE REVIEW

**Comment by teelu** [comment]
I would also like some other alternatives to my animating.  The problem is that the animations flash because I'm removing the left style from it to reset it's position

**Comment by h8rry** [comment]
Only the computer functions works now, right?

**Comment by h8rry** [comment]
It's fun!

File: css/custom.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
+.container {
```
Comment (h8rry) [inline css/custom.css:1]: Fix Indentations. 2 spaces per indentation

File: index.html [hunk 2]
```diff
@@ -1,18 +1,39 @@
 <!DOCTYPE html>
```
Comment (h8rry) [inline index.html:1]: Fix indentations

**Comment by teelu** [comment]
yeah.  only computer.  have to create the functions still to create move objects

**Comment by teelu** [comment]
Indentation fixed and css converted to spaces (don't know how that happened sorry! :O )

**Comment by teelu** [comment]
Merging with latest PR

**Pull request merged**